docs/devices-api/more-info/webhooks.mdx

```diff
@@ -17,15 +16,79 @@ how the Webhook flow works, see [our Webhooks guide](/home/api-info/webhooks).
   page](/devices-api/more-info/our-integrations).
 </Info>
 
-## User data message
+## User provider connection message
```

Comment:
Maybe remove "User" from the title and keep the name more similar to the `type`?

I think its clearer to just state "Provider Connected message"